Bajada de precio a Penetrator

Sombrahide opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
Se ha sobrescrito la variable por defecto del objeto mag_accel para bajar su precio a la hora de exportarlo, evitando de esta manera que cargo se haga rico abusando de su exportacion.
<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game
Previene que la gente de cargo pueda abusar de la venta de Penetrators o por lo menos alivia el problema al hacerlo menos productivo.
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog
:cl:
balance: Se bajo el precio de exportacion del Pentrator de 500 a 400 creditos.
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Sombrahide committed:
Bajada de precio a Penetrator

Se ha sobrescrito la variable por defecto del objeto mag_accel para bajar su precio a la hora de exportarlo, evitando de esta manera que cargo se haga rico abusando de su exportacion.

SagaStars commented:
Increible PR, mis ancestros nunca olvidaran este momento historico.

Silverexperience reviewed (changes requested):
En tu changelog pones que el precio cambia de 500 a 400 pero realmente cambia a 300 creditos.
¿Cual es el precio que le quieres poner?

Silverexperience reviewed (commented):
Aunque bajes el precio si la productividad es alta seguirá siendo una forma muy fácil de enriquecerse.
¿Estás seguro de que la bajada de precio funcionaría?

Silverexperience reviewed (commented):
Para poder ahorrar conflictos al ser un cambio tan pequeño que quizás a los rusos también les interese podrías hacerlo en su repositorio y ya pasaría al nuestro, no veo la necesidad de que este aquí.

Sombrahide commented:
De acuerdo, en ese caso mirare de pasarlo por el suyo, esto sobretodo ha sido una pequeña practica para ver como codear para el server.

Sombrahide commented:
Cierto, perdona que no lo pusiera en el anterior comentario, mas como crees tu que se podria prevenir que se vuelva algo tan beneficioso? Encuentro que siempre debera generar algo de beneficio en comparacion a los recursos gastados, mas no soy un experto en ciencias o cargo para saber cual seria un buen precio para el mismo.

Silverexperience commented:
> 
> 
> Cierto, perdona que no lo pusiera en el anterior comentario, mas como crees tu que se podria prevenir que se vuelva algo tan beneficioso? Encuentro que siempre debera generar algo de beneficio en comparacion a los recursos gastados, mas no soy un experto en ciencias o cargo para saber cual seria un buen precio para el mismo.

Como por lo que veo la mejora es totalmente gratuita de construir, podrías hacer que cueste materiales o que no valiese nada

Silverexperience commented:
Para hacer el PR en su repositorio seria como en este, pero cuando estes en la selección eliges compararlo con el repositorio de discordia y no el de Helix.
![imagen](https://user-images.githubusercontent.com/58746682/90077571-cf8cb680-dd02-11ea-90de-e6f0ae537eb9.png)

Sombrahide commented:
Entendido, muchas gracias Silver, en cuanto termine de colocar los peinados de Ume me pondre a enviar la solicitud a su git

Sombrahide committed:
Ahora los mods de ciencias cuestan material

Mi anterior planteamiento de simplemente bajar el precio al penetrator no ayudaba demasiado a la hora de detener su abuso, al fin y al cabo no costaba nada construirlo, ahora tanto el Penetrator como Caster, los dos mods para armas que puede producir ciencias tienen un costo de produccion de 10 unidades de Plasteel, en caso de que se intente producir y vender se seguiria obteniendo un beneficio de 200 creditos, mas ahora se añadido el trabajo de conseguir primero el Plasteel para producirlo, evitando asi que se spamee de forma continua en el protolathe.

Sombrahide commented:
Understood, I did not realize that someone else had made the change
